app/models/host_status/build_status.rb

```diff
@@ -64,6 +64,21 @@ def token_expired?
     def build_errors?
       host && host.build_errors.present?
     end
+
+    def self.to_label_help(host_status)
+      case host_status
+        when "Pending installation"
+          N_("Wait for sometime and if build fails, rebuild again")
+        when "Token expired"
+          N_("Cancel your build and regenerate token by initiating build again")
+        when "Installed"
+          N_("Done!")
+        when "Installation error"
+          N_("Check audit log")
+        when "Unknown build status"
+          N_("Unknown status")
+      end
+    end
```

Comment:
Why this is a class method? Can't you make this simply an instance method with no parameter?

Also the name should not mention anything from View stack (MVC), let's name it simply `remedation_text` or similar. We can easily use it in non-label context (e.g. in CLI/API).

Which brings us to API part of this patch, we should probably present it there as well.

Reply:
Hello, is it ok to keep this method in helpers or it should be here in model?

Reply:
I believe this class is ideal fit.

Reply:
@lzap @ares Updated the changes.